docs/cluster-configuration.md

```diff
@@ -11,14 +11,13 @@ cluster = Cluster(ClusterConfiguration(
     namespace='default', # Default None
     head_cpus=1, # Default 2
     head_memory=1, # Default 8
-    head_gpus=0, # Default 0
-    num_gpus=0, # Default 0
+    head_extended_resource_requests={'nvidia.com/gpu':0}, # Default 0
```

Comment:
```suggestion
    head_extended_resource_requests={'nvidia.com/gpu':0}, # Default {'nvidia.com/gpu': 0}
```
Maybe to be as clear as possible we should specify that the default is a `nvidia.com/gpu`